services/hooks/src/v1.js

```diff
@@ -354,6 +354,22 @@ builder.declare({
   await this.Hook.remove({hookGroupId, hookId}, true);
   this.publisher.hookDeleted({hookGroupId, hookId});
 
+  try {
+    await this.LastFire.query({
+      hookGroupId: req.params.hookGroupId,
+      hookId: req.params.hookId,
+    }, {
+      handler: async (lastFire) => {
+        await lastFire.remove(false, true);
+      },
+    });
+
+    this.publisher.hookLastFiresDeleted({hookGroupId, hookId});
```

Comment:
I don't think we need to publish a message when this occurs -- nobody really needs to know..